src/components/Bookmark/Bookmark.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import {
+  Alert,
+  ActivityIndicator,
+  Dimensions,
+  Image,
+  ListView,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  View,
+} from 'react-native';
+import { dimensions } from '../Shared/Dimensions';
+import UserUtil from '../../utils/UserUtil';
+import Row from './Row';
+import Text from '../Shared/UniText';
+
+class Bookmark extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = {
+      dataSource: new ListView.DataSource({
+        rowHasChanged: (row1, row2) => row1 !== row2,
+      }),
+      loaded: false,
+      isRefreshing: false,
+      isEmpty: false,
+    };
+  }
+  
+  onRequestCallback(result, error) {
+    if (error) {
+      alert(error);
+    } else if (result) {
+      this.onRequestSuccess(result);
+    }
+  }
+  
+  onRequestSuccess(result) {
+    let bookmarked = result;
+    bookmarked = bookmarked.map((value) => value);
```

Comment:
Result itself contains status code too, so this is for getting arrays in result.